Lazy Evaluation/Lazy Lists/task.md

```diff
@@ -119,8 +119,8 @@ That's why the second argument to `LazyList.cons` is not evaluated at the point
 Instead, it will be evaluated each time someone calls `tail` on a `LazyList` object.
 
 In Scala 2.13, LazyList (previously Stream) became fully lazy from head to tail. To make it possible,
-methods (`filter`, `flatMap`...) are implemented in a way where the head is not being evaluated if is
-not explicitly indicated.
+methods (`filter`, `flatMap`, etc.) are implemented in a way where the head is not evaluated unless it is
+explicitly required.
```

Comment:
I would keep 'indicated' here at the end....
Also, could you please add `` to `LazyList` and `Stream` at the beginning of this paragraph?

Reply:
Good, will do that.